addons/account/models/account_move.py

```diff
@@ -2022,6 +2022,7 @@ def dirty():
                 != values[fname]
                 for fname in values
             )
+            and needed_before.get(key) != values
```

Comment:
You commit message says
> This occurs because the system detect a difference between the recomputed values and the stored data but this will overwrite manually input from user.

The data in the database at that moment is `existing_after[key][fname]`, which is checked above in the `any()`.
You don't have any guarantee that `needed_before` has the values currently in database/cache.

Reply:
@william-andre Thanks for the explanation, I was not sure of my understanding of the whole sync mechanism

I double checked the flow and the tax recomputation occurs when account move lines are created[^1] (i.e. added to the move), so it seems more reasonable to move the tentative fix there

Is it safe to use `skip_invoice_sync`?

[^1]: It seems in `_prepare_tax_totals` we handle manual edition of tax lines, but this is not the case when syncing lines